library/res/color-v21/default_tab_text.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:color="?android:textColorPrimary" android:state_selected="true" />
+  <item android:color="#80FFFFFF" />
```

Comment:
We are not going to be able to generate `android:textColorPrimary ` with 150 alpha this way. So we need to rely in half transparent white. That´s another downside.